fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -1767,40 +1767,40 @@ private static void checkPositive(int value, String desc) {
     }
 
     /**
-     * A builder for the  indexFromIndex policy. Let the caller set a source index and a fallback policy.
+     * A builder for the  indexing policy. Let the caller set a source index and a fallback policy.
      */
-    public static class IndexFromIndexPolicy {
-        public static final IndexFromIndexPolicy INACTIVE = new IndexFromIndexPolicy();
+    public static class IndexingPolicy {
+        public static final IndexingPolicy DEFAULT = new IndexingPolicy();
         @Nullable private final String sourceIndex;
         @Nullable private final Object sourceIndexSubspaceKey; // overrides the sourceIndex
-        private final boolean allowRecordScan;
+        private final boolean forbidRecordScan;
 
         /**
          * Build the index from a source index. Source index must be readable, idempotent, and fully cover the target index.
          * @param sourceIndex source index
          * @param sourceIndexSubspaceKey if non-null, overrides the sourceIndex param
-         * @param allowRecordScan allow fallback to record scan
+         * @param forbidRecordScan forbid fallback to a by-records scan
          */
-        public IndexFromIndexPolicy(@Nullable String sourceIndex, @Nullable Object sourceIndexSubspaceKey, boolean allowRecordScan) {
+        public IndexingPolicy(@Nullable String sourceIndex, @Nullable Object sourceIndexSubspaceKey, boolean forbidRecordScan) {
             this.sourceIndex = sourceIndex;
-            this.allowRecordScan = allowRecordScan;
+            this.forbidRecordScan = forbidRecordScan;
             this.sourceIndexSubspaceKey = sourceIndexSubspaceKey;
         }
 
         /**
          * Build a non-active object.
          */
-        public IndexFromIndexPolicy() {
+        public IndexingPolicy() {
             this.sourceIndex = null;
-            this.allowRecordScan = true;
+            this.forbidRecordScan = false;
```

Comment:
This should probably call `this(null, false, null)` so that there is one true constructor that all the others call.

Reply:
Fixed.